bot/exts/recruitment/talentpool/_review.py

```diff
@@ -41,6 +41,10 @@
     r"(?:<@&\d+> <@&\d+>\n)*?<@!?(\d+?)> \(.+#\d{4}\) for Helper!\n\n\*\*Nominated by:\*\*",
```

Comment:
This regex will have to be updated, although I have no idea how it didn't already break

Reply:
The one I made for threadbot seem to do the job. Does this look better?
```
NOMINATION_MESSAGE_REGEX = re.compile(
    r"<@!?\d+> \((.+)#\d{4}\) for Helper!\n\n\*\*Nominated by:\*\*",
    re.MULTILINE
)
```

Reply:
(ideally when we rewrite this for https://github.com/python-discord/bot/issues/1746 we will store these messages in redis

Reply:
Fixed in https://github.com/python-discord/bot/pull/1928/commits/57288b22e4e42493e2522b86db8a6f55c117d820 as for why it didn't break, this current regex supports both the current style of message, but also the old style before thread-bot. Since there aren't any old-style messages no, we can use the same regex as threadbot does.